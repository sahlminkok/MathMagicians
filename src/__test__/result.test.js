import React from 'react';
import renderer from 'react-test-renderer';
// import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Result from '../components/result';

describe('Result', () => {
  // Jest: tests with snapshots
  test('Result Component renders correctly', () => {
    const tree = renderer.create(<Result />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
