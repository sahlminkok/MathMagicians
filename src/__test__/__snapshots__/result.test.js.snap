// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result Result Component renders correctly 1`] = `
<div
  className="result darkGray"
/>
`;
